app/src/Homepage/Homepage.css

```diff
@@ -55,3 +55,64 @@
   height: 200vh;
   text-align: center;
```

Comment:
<img width="497" alt="Screenshot 2023-11-02 at 5 47 25 PM" src="https://github.com/twang7310/rate-my-cse/assets/145406076/41d072df-ce4e-47be-8bfc-e057d02c6d36">

A couple final things!

change 'height' to 'min-height' so the innerpage underneath grows with the elements
and do the same with .leveltab so the tabs don't get hidden like this

Totally not your fault, but that'd be great

Reply:
Fixed